### DISCUSSION TITLE
오늘 만났을 때 풀었으면 하는 썰

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [ThinkAboutSoftware/AcademicConference#51] [General]
- 멘토님
![image](https://user-images.githubusercontent.com/75019048/123354484-b5686980-d59e-11eb-9b73-98c5c111ce03.png)

그리고..
저의 친구 이야기 ㅋㅋ
왜 이야기를 하기로 했는지 기억은 안나지만(@hi-hyein 님 혹시 이유가 기억나시는지 ㅋㅋ)
 지난 주 모각코에서 이 친구의 경험담을 짧게 풀어보겠습니다!

@jongfeel @yoonucho @hi-hyein

**Comment by hi-hyein** [comment]
<img width="223" alt="스크린샷 2021-06-25 오전 10 20 40" src="https://user-images.githubusercontent.com/29498125/123354615-ffe9e600-d59e-11eb-9037-955963975793.png">
저도 이유는 모르겠는데 메모장에는 적혀있어욬ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by Joseph-Cha** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ세상에 ㅋㅋㅋㅋㅋㅋㅋ
